pagerduty/resource_pagerduty_event_orchestration_path_service.go

```diff
@@ -0,0 +1,621 @@
+package pagerduty
+
+import (
+	"log"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/heimweh/go-pagerduty/pagerduty"
+)
+
+var eventOrchestrationAutomationActionObject = map[string]*schema.Schema{
+	"key": {
+		Type:     schema.TypeString,
+		Required: true,
+	},
+	"value": {
+		Type:     schema.TypeString,
+		Required: true,
+	},
+}
+
+var eventOrchestrationPathServiceCatchAllActions = map[string]*schema.Schema{
```

Comment:
the only difference between catch_all and regular rule actions is the `route_to` attribute, that's why I extracted the schema snippet to a reusable variable